src/components/Footer/Footer.md

```diff
@@ -13,6 +13,6 @@ const [locale, setLocale] = useState('en');
   </Row>
   <Footer 
     currentLgn={locale}
-    setLng={() => setLocale(locale === 'en' ? 'cy' : 'en')} />
+    setLgn={() => setLocale(locale === 'en' ? 'cy' : 'en')} />
```

Comment:
This is slightly trivial, but we're getting a mix of variable names across the project that refer to "language" as variations on `lng` (in the header for instance) and `lgn` (here). I think we should keep consistent and I believe `lng` or `setLng` etc. to be the better contraction of the word "language".

Reply:
I'm not sure how I keep messing up `lng` with `lgn`! But this is fixed with my latest commit